web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -205,35 +206,47 @@ export class BarChartComponent implements AfterViewInit {
 
     this.focus = this.svg
       .append('g')
+      .attr('id', 'focus-bar-chart')
       .attr('transform', 'translate(0, 0)');
 
     // The brush here is an interactive element that allows dragging to display the
     // focus bar chart.
     this.brush = this.svg
       .append('g')
+      .attr('id', 'brush-bar-chart')
       .attr('transform', `translate(0, ${marginBrush.top})`);
 
+    this.boxplot = this.focus
+      .append('g')
+      .attr('id', 'boxplot');
+
     this.xScaleFocus = d3
       .scaleBand()
-      .rangeRound([marginFocus.left, elementWidth - marginFocus.right])
+      .range([marginFocus.left, elementWidth - marginFocus.right])
       .padding(0.1);
     this.yScaleFocus = d3
-      .scaleLinear().range([elementHeight - marginFocus.bottom, marginFocus.top]);
+      .scaleLinear()
+      .range([elementHeight - marginFocus.bottom, marginFocus.top]);
     this.xAxisFocus = this.focus
       .append('g')
-      .attr('class', 'axis axis--x')
+      .attr('class', 'axis axis--xFocus')
       .attr('transform', `translate(0, ${elementHeight - marginFocus.bottom})`);
     this.yAxis = this.focus
       .append('g')
+      .attr('class', 'axis axis--y')
```

Comment:
Also, do you need to have two classes for the axis? Do you even need to select all the elements in the `axis` class?